ShellScript/Bash.sublime-syntax

```diff
@@ -1332,7 +1332,7 @@ contexts:
     # characters ‘\’, ‘$’, and ‘`’.
     - include: string-escapes
     - include: string-interpolations
-    - include: string-commons
```

Comment:
Do we really want to highlight it in heredocs?

If yes, should probably also include the other kinds below.

Reply:
probably we don't, I have now removed it from here :)